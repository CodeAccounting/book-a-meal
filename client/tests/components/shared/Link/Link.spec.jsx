import React from 'react';
import LinkComp from '../../../../src/app/shared/Link';

describe('Link', () => {
  it('renders correctly', () => {
    const shallowWrapper = shallow(<LinkComp clickHandler={jest.fn()}>Click</LinkComp>);

    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
