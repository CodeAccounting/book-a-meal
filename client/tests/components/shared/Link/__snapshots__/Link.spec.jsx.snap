// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<button
  className="link-btn null"
  id={null}
  onClick={[MockFunction]}
>
  Click
</button>
`;
